controllers/categories.js

```diff
@@ -0,0 +1,7 @@
+const {selectCategories} = require('../models/categories');
+
+exports.getCategories = (req, res) => {
+    selectCategories().then((categories) => {
+        res.status(200).send({categories: categories})
```

Comment:
Minor thing, this could simply be {categories} as the key and value has the same name and it will achieve the same effect.